commands/shortenCommand.go

```diff
@@ -34,7 +34,7 @@ func shortenCommand(ctx context.Context, s *discordgo.Session, m *discordgo.Mess
 	*/
 
 	// delete option
-	if params[1] == "delete" {
+	if len(params) > 1 && params[1] == "delete" {
```

Comment:
need to add logic specifically for invalid param length ("no slug specified")

Reply:
![image](https://user-images.githubusercontent.com/30409431/107148455-ff7fe400-694a-11eb-92a2-8b0c2f24b6ab.png)